web-components/src/components/image/index.tsx

```diff
@@ -131,7 +129,7 @@ const Image: React.FC<React.PropsWithChildren<ImageProps>> = ({
             <div
               className={cx(className, classes.background)}
               style={{
-                backgroundImage: `url("${optimizedSrc}"), url("${src}")`,
```

Comment:
when `optimizedSrc` wasn't set (initial state), similarly it would load the non optimized img `src`